<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.8.27">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Publications – Macroevolution &amp; Macroecology</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./images/favicon.png" rel="icon" type="image/png">
<script src="site_libs/quarto-html/quarto.js" type="module"></script>
<script src="site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="site_libs/quarto-html/axe/axe-check.js" type="module"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-ed96de9b727972fe78a7b5d16c58bf87.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-4853f16a337a0a0289494b04c1c3924c.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed quarto-light">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">Macroevolution &amp; Macroecology</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./people.html"> 
<span class="menu-text">People</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./projects.html"> 
<span class="menu-text">Projects</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./publications.html" aria-current="page"> 
<span class="menu-text">Publications</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-full page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    
<!-- main -->
<main class="content column-page" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Publications</h1>
</div>



<div class="quarto-title-meta column-page">

    
  
    
  </div>
  


</header>


<div class="section-block">
<div class="intro-text">
<p>Recent publications from the Macroevolution &amp; Macroecology Group. For full publication lists, see individual Google Scholar profiles.</p>
</div>
<section id="section" class="level2">
<h2 class="anchored" data-anchor-id="section">2026</h2>
<ul>
<li><p>Lindenmayer DB, Florance D, <strong>Scheele</strong> B, Bowd E, Strong C, Macintosh A, Evans MJ (2026) Temporal changes in soil carbon and nitrogen in response to grazing management and vegetation cover in south-eastern Australia. <em>PLoS One</em> 21(2):e0342006. <a href="https://doi.org/10.1371/journal.pone.0342006">doi:10.1371/journal.pone.0342006</a></p></li>
<li><p>Crawford-Ash J et al.&nbsp;incl.&nbsp;<strong>Scheele</strong> BC (2026) Evolution of research on global amphibian declines. <em>Conservation Biology</em> 40(1):e70146. <a href="https://doi.org/10.1111/cobi.70146">doi:10.1111/cobi.70146</a></p></li>
<li><p>Norris E, <strong>Scheele</strong> BC, <strong>Cardillo</strong> M (2026) Land tenure contributions to protected area growth under alternative conservation targets in the Australian monsoon tropics. <em>Conservation Biology</em> 40(1):e70143. <a href="https://doi.org/10.1111/cobi.70143">doi:10.1111/cobi.70143</a></p></li>
<li><p><strong>Hua</strong> X, <strong>Bromham</strong> L, Stewart J, Algy C, Meakins F (2026) Sali-CAT: A new method for ranking social salience for multiple variables. <em>Language Variation and Change</em> in press</p></li>
<li><p><strong>Douglas</strong> J, <strong>Bromham</strong> L (submitted) Reconstructing substitution histories on phylogenies, with accuracy, precision, and coverage. <a href="https://www.biorxiv.org/cgi/content/short/2025.12.21.695861v1">Preprint</a></p></li>
<li><p><strong>Kearns</strong> M, <strong>Skeels</strong> A, Yaxley K, <strong>Cardillo</strong> M (in press) Conservation priority areas based on phylogenetic diversity are sensitive to methods of phylogenetic inference and divergence time estimation. <em>Diversity &amp; Distributions</em></p></li>
</ul>
</section>
<section id="section-1" class="level2">
<h2 class="anchored" data-anchor-id="section-1">2025</h2>
<ul>
<li><p><strong>Bromham</strong> L, Yaxley K, Wilson O, <strong>Hua</strong> X (2025) Macroevolutionary analysis of polysynthesis shows that language complexity is more likely to evolve in small, isolated populations. <em>PNAS</em> 122(24):e2504483122. <a href="https://doi.org/10.1073/pnas.2504483122">doi:10.1073/pnas.2504483122</a></p></li>
<li><p><strong>Hua</strong> X, <strong>Bromham</strong> L (2025) Reply to Koplenig &amp; Wolfer: Global language analyses must properly account for relationships, location and unbalanced binary data. <em>PNAS</em> 122(41):e2518837122. <a href="https://doi.org/10.1073/pnas.2518837122">doi:10.1073/pnas.2518837122</a></p></li>
<li><p><strong>Skeels</strong> A, Sauquet H, Mast A, Weston PH, Olde PM, Jordan GJ, Carpenter RJ, Fenker J, Reynolds ZKM, Lemmon AR, Lemmon EM, Marcano FP, <strong>Cardillo</strong> M (2025) Paleobiome dynamics shaped a large Gondwanan plant radiation. <em>PNAS</em> 122(29):e2502129122. <a href="https://doi.org/10.1073/pnas.2502129122">doi:10.1073/pnas.2502129122</a></p></li>
<li><p>Duncan RP, <strong>Scheele</strong> BC, Clulow S (2025) Thermal mismatch models derived from occurrence data predict pathogen prevalence in frogs. <em>PNAS</em> 122(30):e2423706122. <a href="https://doi.org/10.1073/pnas.2423706122">doi:10.1073/pnas.2423706122</a></p></li>
<li><p>Ward M et al.&nbsp;incl.&nbsp;<strong>Scheele</strong> BC (2025) The estimated cost of preventing extinction and progressing recovery for Australia’s priority threatened species. <em>PNAS</em> 122(6):e2414985122. <a href="https://doi.org/10.1073/pnas.2414985122">doi:10.1073/pnas.2414985122</a></p></li>
<li><p><strong>Cardillo</strong> M, <strong>Scheele</strong> BC, Tulloch AIT (2025) Forecasting extinction risk for future-proof conservation decisions. <em>Trends in Ecology &amp; Evolution</em>. <a href="https://doi.org/10.1016/j.tree.2025.10.014">doi:10.1016/j.tree.2025.10.014</a></p></li>
<li><p>Buettel JC, Lindenmayer DB, <strong>Scheele</strong> BC, Evans MJ (2025) Maintaining robust terrestrial ecological monitoring amid technological advancements. <em>Trends in Ecology &amp; Evolution</em>. <a href="https://doi.org/10.1016/j.tree.2025.04.003">doi:10.1016/j.tree.2025.04.003</a></p></li>
<li><p>Schembri R, <strong>Bromham</strong> L, Moritz C, <strong>Hua</strong> X (2025) Comparing rates of molecular and morphological evolution identifies multiple speciation trajectories in a diverse radiation of skinks. <em>Evolution</em> 79:858–868. <a href="https://doi.org/10.1093/evolut/qpaf033">doi:10.1093/evolut/qpaf033</a></p></li>
<li><p><strong>Hua</strong> X, Moritz C (2025) A phylogenetic approach to delimitate species in a probabilistic way. <em>Systematic Biology</em> 74(5):723–739. <a href="https://doi.org/10.1093/sysbio/syaf004">doi:10.1093/sysbio/syaf004</a></p></li>
<li><p><strong>Bromham</strong> L (2025) Population size and language change: an evolutionary perspective. <em>Annual Review of Linguistics</em> 11:183–208. <a href="https://doi.org/10.1146/annurev-linguistics-031422-123959">doi:10.1146/annurev-linguistics-031422-123959</a></p></li>
<li><p><strong>Bromham</strong> L (2025) The genotype concept and language evolution. <em>Physics of Life Reviews</em> 52:23–26. <a href="https://doi.org/10.1016/j.plrev.2024.11.014">doi:10.1016/j.plrev.2024.11.014</a></p></li>
<li><p>Ding W, Ree RH, May MR, Brun P, Hagen O, Karger DN, <strong>Skeels</strong> A et al.&nbsp;(2025) The asynchronous rise of Northern Hemisphere alpine floras reveals general responses of biotic assembly to orogeny and climate change. <em>Science Advances</em> 11(51):eadz1888. <a href="https://doi.org/10.1126/sciadv.adz1888">doi:10.1126/sciadv.adz1888</a></p></li>
<li><p>Nge FJ, <strong>Skeels</strong> A (2025) Diversification patterns of the southwest Australian biodiversity hotspot reveal a novel macroevolutionary pathway to plant hyperdiversity. <em>New Phytologist</em> 248(2):953–967. <a href="https://doi.org/10.1111/nph.70330">doi:10.1111/nph.70330</a></p></li>
<li><p>Yuan Z, Yang G, Ding W, Brun P, <strong>Skeels</strong> A, Pellissier L, Zimmermann NE (2025) The geography of connectivity shapes plant endemism hotspots. <em>Ecography</em> 2025(9):e07514. <a href="https://doi.org/10.1002/ecog.07514">doi:10.1002/ecog.07514</a></p></li>
<li><p>Lorcery M, Husson L, Salles T, Lavergne S, Hagen O, <strong>Skeels</strong> A (2025) Deep time evolution of the Latitudinal Diversity Gradient: Insights from mechanistic models. <em>PLoS One</em> 20(9):e0332766. <a href="https://doi.org/10.1371/journal.pone.0332766">doi:10.1371/journal.pone.0332766</a></p></li>
<li><p>Esquerré D, <strong>Skeels</strong> A (2025) Biogeographic Patterns of Richness and Endemism in Liolaemidae: Identifying Hotspots for the Most Diverse Andean Reptile Radiation. In <em>Andean Herpetofauna: Explorations of Diversity, Ecology, and Conservation</em>. <a href="https://doi.org/10.1007/978-3-032-00074-3_5">doi:10.1007/978-3-032-00074-3_5</a></p></li>
<li><p>Chowdhury S, <strong>Cardillo</strong> M et al.&nbsp;(2025) Protected area coverage of the full annual cycle of migratory butterflies. <em>Conservation Biology</em> 39(3):e14423. <a href="https://doi.org/10.1111/cobi.14423">doi:10.1111/cobi.14423</a></p></li>
<li><p>Odebiri O, <strong>Scheele</strong> BC, Lindenmayer DB, Smith DG, Malerba ME (2025) Fencing the Flux: Seasonal Trends, Environmental Drivers, and Mitigation Opportunities of Methane Emissions From Farm Dams. <em>Global Change Biology</em> 31(12):e70637. <a href="https://doi.org/10.1111/gcb.70637">doi:10.1111/gcb.70637</a></p></li>
<li><p><strong>Scheele</strong> BC, Heard GW, Duncan RP, Clulow S, Sopniewski J (2025) An invasive pathogen generally contracts species to their niche cores, not margins. <em>Ecography</em> 2025(5):e07612. <a href="https://doi.org/10.1111/ecog.07612">doi:10.1111/ecog.07612</a></p></li>
<li><p>Crawford-Ash J et al.&nbsp;incl.&nbsp;<strong>Scheele</strong> BC (2025) Defying decline: Very low chytrid prevalence in tadpoles, yet high infection in adults in a naturally recovering frog species. <em>Animal Conservation</em> 28(4):567–581. <a href="https://doi.org/10.1111/acv.13006">doi:10.1111/acv.13006</a></p></li>
<li><p>Smith DG, Evans MJ, <strong>Scheele</strong> BC et al.&nbsp;(2025) Grazing control and revegetation increases bird biodiversity at farm dams. <em>Biological Conservation</em> 309:111310. <a href="https://doi.org/10.1016/j.biocon.2025.111310">doi:10.1016/j.biocon.2025.111310</a></p></li>
<li><p>Lindenmayer D, <strong>Scheele</strong> B et al.&nbsp;(2025) Declining trajectories characterise arboreal marsupial assemblages in eastern Australia. <em>Biological Conservation</em> 308:111264. <a href="https://doi.org/10.1016/j.biocon.2025.111264">doi:10.1016/j.biocon.2025.111264</a></p></li>
<li><p>Ward M et al.&nbsp;incl.&nbsp;<strong>Scheele</strong> BC (2025) Half of the habitat of Australia’s highly imperilled narrow-range species is outside protected areas. <em>Biological Conservation</em> 308:111195. <a href="https://doi.org/10.1016/j.biocon.2025.111195">doi:10.1016/j.biocon.2025.111195</a></p></li>
<li><p>Jones SKC et al.&nbsp;incl.&nbsp;<strong>Scheele</strong> BC (2025) Quantifying taxon-specific habitat connectivity requirements of urban wildlife using structured expert judgement. <em>Biological Conservation</em> 305:110991. <a href="https://doi.org/10.1016/j.biocon.2025.110991">doi:10.1016/j.biocon.2025.110991</a></p></li>
<li><p>Bell K, Evans MJ, Lindenmayer DB, <strong>Scheele</strong> BC et al.&nbsp;(2025) Excluding livestock from farm dams enhances native biodiversity. <em>Agriculture, Ecosystems &amp; Environment</em> 386:109623. <a href="https://doi.org/10.1016/j.agee.2025.109623">doi:10.1016/j.agee.2025.109623</a></p></li>
<li><p>Hingee KL, <strong>Scheele</strong> BC et al.&nbsp;(2025) Model-based estimates of bird diversity reveal benefits from disparate, landscape-scale restoration activities. <em>Restoration Ecology</em> e70271. <a href="https://doi.org/10.1111/rec.70271">doi:10.1111/rec.70271</a></p></li>
<li><p>Taylor M, <strong>Scheele</strong> BC et al.&nbsp;(2025) Effects of fire severity and fire frequency on the abundance of hollows and hollow-bearing trees. <em>Forest Ecology and Management</em> 596:123110. <a href="https://doi.org/10.1016/j.foreco.2025.123110">doi:10.1016/j.foreco.2025.123110</a></p></li>
<li><p>Littlefair M, <strong>Scheele</strong> BC et al.&nbsp;(2025) Enhancing Farm Dams Increases Tadpole Abundance. <em>Ecology and Evolution</em> 15(1):e70803. <a href="https://doi.org/10.1002/ece3.70803">doi:10.1002/ece3.70803</a></p></li>
<li><p>Mangan MJ, McCallum HI, West M, <strong>Scheele</strong> BC, Gillespie GR, Grogan LF (2025) Differential recruitment drives pathogen-mediated competition between species in an amphibian chytridiomycosis system. <em>Ecological Applications</em> 35(1):e3085. <a href="https://doi.org/10.1002/eap.3085">doi:10.1002/eap.3085</a></p></li>
<li><p>Jofré P, Aguilera-Gómez C, Villarreal P, Cubillos FA, Das P, <strong>Hua</strong> X et al.&nbsp;(2025) Studying stellar populations in Omega Centauri with phylogenetics. <em>Astronomy &amp; Astrophysics</em> 699:A291. <a href="https://doi.org/10.1051/0004-6361/202554863">doi:10.1051/0004-6361/202554863</a></p></li>
</ul>
</section>
<section id="section-2" class="level2">
<h2 class="anchored" data-anchor-id="section-2">2024</h2>
<ul>
<li><p><strong>Bromham</strong> L, Yaxley KJ, <strong>Cardillo</strong> M (2024) Islands are engines of language diversity. <em>Nature Ecology and Evolution</em> 8:1991–2002. <a href="https://www.nature.com/articles/s41559-024-02488-4">doi:10.1038/s41559-024-02488-4</a></p></li>
<li><p>Langmore NE, Grealy A, Noh HJ, Medina I, <strong>Skeels</strong> A, Grant J, Murray KD et al.&nbsp;(2024) Coevolution with hosts underpins speciation in brood-parasitic cuckoos. <em>Science</em> 384(6699):1030–1036. <a href="https://doi.org/10.1126/science.adj3210">doi:10.1126/science.adj3210</a></p></li>
<li><p><strong>Bromham</strong> L (2024) Mutation rate is central to understanding evolution. <em>American Journal of Botany</em> 111(10):e16422. <a href="https://doi.org/10.1002/ajb2.16422">doi:10.1002/ajb2.16422</a></p></li>
<li><p><strong>Bromham</strong> L (2024) Combining Molecular, Macroevolutionary, and Macroecological Perspectives on the Generation of Diversity. <em>Cold Spring Harbor Perspectives in Biology</em> 19:a041453. <a href="https://doi.org/10.1101/cshperspect.a041453">doi:10.1101/cshperspect.a041453</a></p></li>
<li><p><strong>Bromham</strong> L (2024) Solving Galton’s problem: practical solutions for analysing language diversity and evolution. <em>Current Issues in Linguistic Theory</em> 367:74–108. <a href="https://doi.org/10.1075/cilt.367.04bro">doi:10.1075/cilt.367.04bro</a></p></li>
<li><p>Meakins F, <strong>Bromham</strong> L, <strong>Hua</strong> X (2024) Depending on gender: The role of Gurindji women in contact-induced language change. <em>Asia-Pacific Language Variation</em> 10:183–208. <a href="https://doi.org/10.1075/aplv.24001.mea">doi:10.1075/aplv.24001.mea</a></p></li>
<li><p>Moroti MT, <strong>Skeels</strong> A, da Silva FR, Provete DB (2024) Climate Interacts With Diversification Rate in Determining Species Richness and Trait Diversity of Tetrapods in a Global Hotspot. <em>Journal of Biogeography</em> 51(12):2484–2497. <a href="https://doi.org/10.1111/jbi.15001">doi:10.1111/jbi.15001</a></p></li>
<li><p>Tiatragul S, <strong>Skeels</strong> A, Keogh JS (2024) Morphological evolution and niche conservatism across a continental radiation of Australian blindsnakes. <em>Evolution</em> 78(11):1854–1868. <a href="https://doi.org/10.1093/evolut/qpae132">doi:10.1093/evolut/qpae132</a></p></li>
<li><p><strong>Skeels</strong> A et al.&nbsp;incl.&nbsp;<strong>Cardillo</strong> M (2024) Sources of gene tree discordance and their implications for systematics and evolution of a megadiverse Australian plant radiation. <em>Bulletin of the Society of Systematic Biologists</em> 3(2). <a href="https://doi.org/10.18061/bssb.v3i2.9823">doi:10.18061/bssb.v3i2.9823</a></p></li>
<li><p>Lindenmayer D, <strong>Scheele</strong> BC et al.&nbsp;(2024) Multiple Long-Term, Landscape-Scale Data Sets Reveal Intraspecific Spatial Variation in Temporal Trends for Bird Species. <em>Ecology Letters</em> 27(12):e14531. <a href="https://doi.org/10.1111/ele.14531">doi:10.1111/ele.14531</a></p></li>
<li><p>Coates J, Evans M, <strong>Scheele</strong> B, Encinas-Viso F, Florez Fernandez J et al.&nbsp;(2024) Influence of climate, weather and floral associations on pollinator community composition across an elevational gradient. <em>Oikos</em> 2024(12):e10688. <a href="https://doi.org/10.1111/oik.10688">doi:10.1111/oik.10688</a></p></li>
<li><p>Sopniewski J, Catullo R, Ward M, Mitchell N, <strong>Scheele</strong> BC (2024) Niche-based approach to explore the impacts of environmental disturbances on biodiversity. <em>Conservation Biology</em> 38(6):e14277. <a href="https://doi.org/10.1111/cobi.14277">doi:10.1111/cobi.14277</a></p></li>
<li><p>Heard GW, Scroggie MP, Hollanders M, <strong>Scheele</strong> BC (2024) Age truncation due to disease shrinks metapopulation viability for amphibians. <em>Journal of Animal Ecology</em> 93(11):1670–1683. <a href="https://doi.org/10.1111/1365-2656.14177">doi:10.1111/1365-2656.14177</a></p></li>
<li><p><strong>Scheele</strong> BC, Webb RJ, <strong>Hua</strong> X, Hollanders M (2024) Variation in amphibian maturation rates influences population vulnerability to disease-induced declines. <em>Animal Conservation</em> 27(5):600–610. <a href="https://doi.org/10.1111/acv.12939">doi:10.1111/acv.12939</a></p></li>
<li><p>Evans MJ, Beggs R, <strong>Scheele</strong> BC et al.&nbsp;(2024) Farm dam enhancement significantly improves water quality. <em>Agriculture, Ecosystems &amp; Environment</em> 373:109134. <a href="https://doi.org/10.1016/j.agee.2024.109134">doi:10.1016/j.agee.2024.109134</a></p></li>
<li><p>Littlefair M, <strong>Scheele</strong> BC et al.&nbsp;(2024) Management to enhance farm dam condition improves outcomes for amphibians. <em>Agriculture, Ecosystems &amp; Environment</em> 374:109156. <a href="https://doi.org/10.1016/j.agee.2024.109156">doi:10.1016/j.agee.2024.109156</a></p></li>
<li><p>Smith DG, Evans MJ, <strong>Scheele</strong> BC, Crane M, Lindenmayer DB (2024) Co-location of multiple natural assets on farms increases bird species richness and breeding activity. <em>Agriculture, Ecosystems &amp; Environment</em> 359:108765. <a href="https://doi.org/10.1016/j.agee.2023.108765">doi:10.1016/j.agee.2023.108765</a></p></li>
<li><p>Littlefair M, <strong>Scheele</strong> BC, Westgate M, Lindenmayer D (2024) The ecological and biodiversity conservation values of farm dams: A systematic review. <em>PLoS One</em> 19(5):e0303504. <a href="https://doi.org/10.1371/journal.pone.0303504">doi:10.1371/journal.pone.0303504</a></p></li>
<li><p>Woinarski JCZ et al.&nbsp;incl.&nbsp;<strong>Scheele</strong> BC (2024) Few havens for threatened Australian animal taxa that are highly susceptible to introduced and problematic native species. <em>Biodiversity and Conservation</em> 33(1):305–331. <a href="https://doi.org/10.1007/s10531-023-02750-4">doi:10.1007/s10531-023-02750-4</a></p></li>
<li><p>Walsen K, Jofré P, Buder S, Yaxley K, Das P, Yates RM, <strong>Hua</strong> X, Signor T et al.&nbsp;(2024) Assembling a high-precision abundance catalogue of solar twins in GALAH for phylogenetic studies. <em>Monthly Notices of the Royal Astronomical Society</em> 529(3):2946–2966. <a href="https://doi.org/10.1093/mnras/stae280">doi:10.1093/mnras/stae280</a></p></li>
<li><p>de Brito Silva D, Jofré P, Tissera PB, Yaxley KJ, Gonzalez Jara J, <strong>Hua</strong> X et al.&nbsp;(2024) On the Evolutionary History of a Simulated Disk Galaxy as Seen by Phylogenetic Trees. <em>The Astrophysical Journal</em> 962(2):154. <a href="https://doi.org/10.3847/1538-4357/ad036a">doi:10.3847/1538-4357/ad036a</a></p></li>
</ul>
</section>
</div>



</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
      const outerScaffold = trigger.parentElement.cloneNode(true);
      const codeEl = outerScaffold.querySelector('code');
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      &nbsp;
    </div>   
    <div class="nav-footer-center">
<p>Macroevolution &amp; Macroecology Group | Research School of Biology | Australian National University</p>
</div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>




</body></html>